src/components/ThumbsFeedback/ThumbsFeedback.tsx

```diff
@@ -0,0 +1,165 @@
+      }
+    });
+  };
+
+  const handleSubmit = async (event: FormEvent<HTMLFormElement>): Promise<void> => {
+    event.preventDefault();
+
+    if (!isValidCommentLength(comment, MAX_COMMENT_LENGTH)) {
+      return;
+    }
+
+    const trimmedComment = comment.trim();
+
+    if (!containsPII(trimmedComment)) {
+      trackEvent({
+        event_name: 'feedback_comment_submit',
+        event_category: 'feedback',
+        event_label: `comment_thumbs_${feedback}`,
+        custom_parameters: {
+          comment_text: trimmedComment,
+          thumb_direction: feedback
+        }
+      });
+    }
+
+    setIsSubmitted(true);
+  };
+
+  if (isSubmitted) {
+    return <p>Thank you for your feedback.</p>;
+  }
+
+  const shouldShowFeedback: boolean = !isSidebarScrollable;
+
+  if (!shouldShowFeedback) {
+    return null;
+  }
+
+  return (
+    <div className={styles.thumbsFeedback}>
+      <form onSubmit={handleSubmit}>
+        <p id="feedback" className={styles.feedbackTitle}>
+          Was this page helpful?
+        </p>
+        {showButtons ? (
+          <div className={styles.svgContainer}>
+            <span
+              className={styles.thumbsUp}
+              style={{ cursor: "pointer" }}
+              onClick={() => handleFeedbackClick("up")}
+              tabIndex={0}
+              role="button"
+              aria-label="Thumbs up"
+            >
+              <Icon name="thumbsUp" size="md" />
+            </span>
+            <span
+              className={styles.thumbsDown}
+              style={{ cursor: "pointer" }}
+              onClick={() => handleFeedbackClick("down")}
+              tabIndex={0}
+              role="button"
+              aria-label="Thumbs down"
+            >
+              <Icon name="thumbsDown" size="md" />
+            </span>
+          </div>
+        ) : (
+          <div>
+              <textarea
+                id="comment"
+                name="comment"
+                value={comment}
+                placeholder="Any additional comments..."
+                onChange={(e) => setComment(e.target.value)}
+                className={`${styles.commentTextarea} ${comment.length > MAX_COMMENT_LENGTH ? styles.error : ''}`}
+              />
+              <div className={`${styles.characterCount} ${comment.length > MAX_COMMENT_LENGTH ? styles.error : ''}`}>
+                ({comment.length}/{MAX_COMMENT_LENGTH}) characters allowed
+              </div>
+              <div className={styles.submitButton}>
+                <Button
+                  type="submit"
+                  as="button"
+                  variant="primary"
+                  disabled={comment.length > MAX_COMMENT_LENGTH}
+                >
+                  Submit
+                </Button>
+                <p className={styles.feedbackTitle}> or </p>
+                <div className={styles.githubLinkWrapper}>
+                  <GitHubIssueLink pathname={location.pathname} />
```

Comment:
Might make sense to adjust the font here to work better with the surrounding content!